### PR TITLE
Update tests and linting 

### PR DESCRIPTION
**Pull request by Quentinchampenois**
#### :tophat: What? Why?

Some tests were out of date after accessibility updates.
Run linter for JS, CSS, ERB and rubocop. 

#### :clipboard: Subtasks
- [x] Update tests according to accessibility updates
- [x] Add  some tests for icon helper updates
- [x] Linting css, js, erb and rubocop
- [x] Translation normalization

### CODE REVIEW

File: decidim-core/app/views/layouts/decidim/_logo.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <% if organization %>
-  <%= link_to decidim.root_url(host: organization.host), alt: t("logo_alt", organization: organization.name, scope: "decidim.accessibility") do %>
+  <%= link_to decidim.root_url(host: organization.host), alt: t("decidim.accessibility.logo_alt", organization: organization.name) do %>
```
Comment (armandfardeau) [inline decidim-core/app/views/layouts/decidim/_logo.html.erb]: Hi,
Have you tried the following?
```suggestion
  <%= link_to decidim.root_url(host: organization.host), alt: t("logo_alt", scope: "decidim.accessibility", organization: organization.name) do %>
```

File: decidim-assemblies/spec/system/assemblies_spec.rb [hunk 2]
```diff
@@ -105,7 +105,7 @@
 
     it "lists the parent assemblies" do
       within "#parent-assemblies" do
-        within "#parent-assemblies h2" do
+        within "#parent-assemblies h3" do
```
[the remark this answers is not in the document]
Comment (Quentinchampenois) [reply decidim-assemblies/spec/system/assemblies_spec.rb]: Hi, I checked the font size and it seems to be the same !

File: decidim-comments/app/frontend/comments/comments.component.test.tsx [hunk 3]
```diff
@@ -91,7 +91,7 @@ describe("<Comments />", () => {
   it("renders comments count", () => {
     const wrapper = shallow(<Comments commentable={commentable} session={session} reorderComments={reorderComments} orderBy={orderBy} />);
     const rex = new RegExp(`${commentable.comments.length} comments`);
-    expect(wrapper.find("h2.section-heading").text()).toMatch(rex);
+    expect(wrapper.find("h3.section-heading").text()).toMatch(rex);
```
[the remark this answers is not in the document]
Comment (Quentinchampenois) [reply decidim-comments/app/frontend/comments/comments.component.test.tsx]: section-heading class defines a font-size

File: decidim-core/app/views/layouts/decidim/_logo.html.erb (hunk 1 above)
Comment (Quentinchampenois) [reply decidim-core/app/views/layouts/decidim/_logo.html.erb]: Thanks for your proposition, it's interesting. I try it

File: decidim-core/app/assets/stylesheets/decidim/layouts/_user.scss [hunk 4]
```diff
@@ -49,4 +49,12 @@
     margin-right: .7em;
     vertical-align: middle;
   }
+
+  h1{
```
Comment (MoretS) [inline decidim-core/app/assets/stylesheets/decidim/layouts/_user.scss]: Are you sure about red color for h1 ?

Comment (MoretS) [inline decidim-core/app/assets/stylesheets/decidim/layouts/_user.scss]:
```suggestion
```

File: decidim-core/app/assets/stylesheets/decidim/layouts/_user.scss [hunk 5]
```diff
@@ -49,4 +49,12 @@
...
+    color: red;
```
Comment (MoretS) [inline decidim-core/app/assets/stylesheets/decidim/layouts/_user.scss]:
```suggestion
```

File: decidim-core/app/assets/stylesheets/decidim/layouts/_user.scss [hunk 6]
```diff
@@ -49,4 +49,12 @@
...
+  }
```
Comment (MoretS) [inline decidim-core/app/assets/stylesheets/decidim/layouts/_user.scss]:
```suggestion
```

File: .circleci/config.yml [hunk 7]
```diff
@@ -12,6 +12,7 @@ defaults: &defaults
     - image: postgres
       environment:
         POSTGRES_USER: postgres
+	POSTGRES_HOST_AUTH_METHOD: trust
```
Comment (armandfardeau) [inline .circleci/config.yml]: I don't think indentation going to work here.

**Comment by moustachu** [comment]
Not needed but GO ! 👍

**Comment by armandfardeau** [comment]
> Not needed but GO ! 👍

It's needed since 4 days, due to a docker image update

File: .circleci/config.yml (hunk 7 above)
Comment (Quentinchampenois) [reply .circleci/config.yml]: Thanks you, you are right. I refactored it

**Pull request merged**